### DISCUSSION TITLE
Run on MQTT stream

### DISCUSSION THREAD

**Discussion by bgro82** [Motion-Project/motion#1658] [General]
Is there a way to use motion for MQTT camera streams? I couldn't find anything about this searching around but wondering if someone has figured out some clever way I'm not realizing to do this. Thank you!

**Comment by tosiara** [comment]
Are you able to view MQTT stream using ffplay or vlc?

**Comment by bgro82** [comment]
Unfortunately not as far as I can tell which I presume means it isn't easily done. Thank you!
